.eslintrc.yml

```diff
@@ -1,138 +1,66 @@
+  jsx-a11y/no-static-element-interactions: off
+
+  array-bracket-spacing:
+    - error
+    - always
   brace-style:
     - error
     - stroustrup
-  camelcase: off
-  consistent-return: error
-  curly:
-    - error
-    - all
-  comma-dangle:
-    - error
-    - always-multiline
-  dot-notation:
+  import/no-unresolved:
     - error
     -
-      allowKeywords: true
-  eqeqeq:
-    - error
-    - allow-null
-  space-before-blocks: error
-  strict: off
-  new-cap: error
-  no-bitwise: error
-  no-caller: error
-  no-case-declarations: off
-  no-cond-assign:
+      commonjs: true
+      caseSensitive: true
+      ignore: [^~]
+  no-param-reassign:
     - error
-    - except-parens
-  no-debugger: error
-  no-empty: error
-  no-eval: error
-  no-extend-native: error
-  no-irregular-whitespace: error
-  no-iterator: error
-  no-multi-str: error
-  no-new: error
-  no-proto: error
-  no-script-url: error
-  no-sequences: error
-  no-undef: error
+    -
+      props: false
   no-unused-vars:
     - error
-    - args: none
-  no-with: error
-  one-var:
-    - error
-    - never
-  indent:
-    - error
-    - tab
     -
-      SwitchCase: 1
-  keyword-spacing: error
-  object-curly-spacing:
+      args: none
+  object-curly-newline:
     - error
-    - always
-  quotes:
-    - error
-    - single
-  semi:
-    - error
-    - always
-  space-in-parens: error
-  valid-typeof: error
-  wrap-iife:
-    - error
-    - inside
-  array-bracket-spacing:
-    - error
-    - always
     -
-      objectsInArrays: false
-      arraysInArrays: false
-
-  no-warning-comments: warn
-  no-console: warn
-
-  no-var: error
-  arrow-spacing: error
-  prefer-arrow-callback: error
-  prefer-const: error
-  prefer-rest-params: error
-  prefer-spread: error
-  prefer-template: error
-  rest-spread-spacing: error
-  template-curly-spacing: error
-
-
-  react/prop-types:
+      multiline: true
+      consistent: true
+  react/default-props-match-prop-types:
     - error
     -
-      skipUndeclared: true
-  react/no-typos: error
-  react/no-string-refs: error
-  react/no-unknown-property: error
-  react/no-unused-prop-types: error
-  react/no-unused-state: error
-  react/self-closing-comp: error
-  react/sort-comp: error
-  react/style-prop-object: error
-  react/void-dom-elements-no-children: error
-  react/jsx-key: error
-  react/jsx-pascal-case: error
-  react/jsx-no-duplicate-props: error
-  react/jsx-closing-bracket-location: error
-  react/jsx-closing-tag-location: error
-  react/jsx-curly-spacing: error
-  react/jsx-equals-spacing: error
-  react/jsx-tag-spacing: error
-  react/jsx-first-prop-new-line:
+      allowRequiredDefaults: true
+  react-hooks/rules-of-hooks:
     - error
-    - multiline
-  react/jsx-wrap-multilines:
+  spaced-comment:
     - error
+    - always
     -
-      declaration: true
-      assignment: true
-      return: true
-      arrow: true
+      exceptions: [ "!steal-remove-start", "!steal-remove-end" ]
```

Comment:
I will not use something that modified code for me, so we cannot use stock prettier. It does not look like the prettier rules are released in any other way besides that tool.

Before you ask, `standard` is not standard and I will not let go of semicolons.

As noted above, this project used to use a fully custom eslint config, and i switched it to use airbnb with some changes. I can revisit how many of those changes are just preferences and how many I think are necessary, in order to reduce the differences, but a few (like the one you specifically tagged here, because this is being built with steal) will have to stay.

Reply:
> I will not use something that modified code for me
Why not? Isn't that kind of the point?

We could go with [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier)
That way, anyone cool with it can let their editor auto fix, and those who don't like tools changing their code can just fix the issues themselves